.github/workflows/build-vscode-extension.yml

```diff
@@ -78,7 +78,8 @@ jobs:
           npm install
 
           cd microsoft-kiota
-          npm install ../npm-package/microsoft-kiota-1.0.0.tgz
+          tgz_file=$(ls ../npm-package/*.tgz)
```

Comment:
we should not need to have to do that if in a proper workspace configuration.
Also does that means we need to run that locally when debugging the extension? This would seriously degrade the experience.

Reply:
let's keep this conversation open until this is actually resolved

Reply:
This has been resolved